backend/btrixcloud/storages.py

```diff
@@ -454,24 +456,27 @@ async def get_presigned_url(
 
         s3storage = self.get_org_storage_by_ref(org, crawlfile.storage)
 
-        async with self.get_s3_client(s3storage, s3storage.use_access_for_presign) as (
-            client,
-            bucket,
-            key,
-        ):
+        async with self.get_s3_client(
+            s3storage,
+            True,
```

Comment:
```suggestion
            for_presign=True,
```
Just to make it a bit more readable later on without having to go look up the signature.